core/platform/cloud_translate/cloud_translate_services.py

```diff
@@ -0,0 +1,54 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides translate_text functionality from Google Cloud Translate."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import utils
+
+# To use cloud translate in a local dev environment, either use
+# cloud_translate_emulator, or follow the setup instructions here:
+# https://cloud.google.com/translate/docs/setup#python
+# Once you have completed the "Using the service account key file in your
+# environment" instructions, google.cloud.translate_v2 can be used by the local
+# server with no further setup.
+from google.cloud import translate_v2 as translate
+
+CLIENT = translate.Client()
+
+
+def translate_text(text, source_language, target_language):
+    """Translates text into the target language.
+
+    Args:
+        text: str. The text to be translated.
+        source_language: str. A valid ISO 639-1 language code.
+        target_language: str. A valid ISO 639-1 language code.
```

Comment:
Please add a blurb or link explaining what ISO 639-1 is to save readers the time of looking it up themselves.

Reply:
I added a link.
This has also been updated to reflect the use of a language code whitelist.